bootstrap/sql/mysql/create_tables.sql

```diff
@@ -352,3 +352,31 @@ CREATE TABLE IF NOT EXISTS component (
   timestamp BIGINT,
   PRIMARY KEY (id)
 );
+
+CREATE TABLE IF NOT EXISTS topology_test_run_case (
+  id BIGINT AUTO_INCREMENT NOT NULL,
+  topologyId BIGINT NOT NULL,
+  versionId BIGINT NOT NULL,
+  records TEXT NOT NULL,
+  timestamp BIGINT,
+  PRIMARY KEY (id),
+  FOREIGN KEY (topologyId) REFERENCES topology(id),
+  FOREIGN KEY (versionId) REFERENCES topology_version(id)
+);
+
+CREATE TABLE IF NOT EXISTS topology_test_run_histories (
+  id BIGINT AUTO_INCREMENT NOT NULL,
+  topologyId BIGINT NOT NULL,
```

Comment:
Maybe we could (and might be better to) replace our own implementation of ORM with popular and well defined thing like Hibernate. I'm not sure it can cover both RDB and Phoenix though.